python/data/provider.py

```diff
@@ -124,7 +124,7 @@ def get_t_end_us(self):
     def get_events_until(self, time: int) -> Optional[Events]:
         # Returns events if successful, None otherwise.
 
-        assert time > self.last_time, f'time = {time}, last_time={self.last_time}'
+        assert time >= self.last_time, f'time = {time}, last_time={self.last_time}'
```

Comment:
why the change?

Reply:
because two events could have the same timestamp, correct?

Reply:
ok, I think it does not matter and it's fine like this

Reply:
all right. thanks for your comment